src/components/MiniCart/MiniCart.js

```diff
@@ -70,16 +71,18 @@ const MiniCart = () => {
             );
           })}
         </MiniCartItems>
-        <CheckoutButton
-          onClick={() => {
-            if (typeof window !== "undefined") {
-              storeDispatch({ type: "toggleMiniCart" });
-              window.location = "/checkout/customer";
-            }
-          }}
-        >
-          Checkout
-        </CheckoutButton>
+        <ButtonContainer>
+          <CheckoutButton
+            onClick={() => {
+              if (typeof window !== "undefined") {
+                storeDispatch({ type: "toggleMiniCart" });
+                window.location = "/cart";
```

Comment:
Instead of `window.location` you can use **reach/router** navigate api https://reach.tech/router/api/navigate

```suggestion
                navigate("/cart ");
```

and: 
```js
import { navigate } from "@reach/router"
````
in import section